dist/stat_data.py

```diff
@@ -311,6 +311,13 @@ def __init__(self, name, desc, flags=''):
             'no_clear,no_scale'),
     TxnStat('txn_pinned_range',
         'transaction range of IDs currently pinned', 'no_clear,no_scale'),
+    TxnStat('txn_pinned_snapshot_range',
+        'transaction range of IDs currently pinned by named snapshots',
+            'no_clear,no_scale'),
+    TxnStat('txn_snapshots_created',
+        'number of named snapshots created', 'no_clear,no_scale'),
+    TxnStat('txn_snapshots_dropped',
+        'number of named snapshots dropped', 'no_clear,no_scale'),
```

Comment:
I didn't expect these counts of operations to be "no_clear,no_scale" -- was that intentional?